variables.tf

```diff
@@ -112,8 +160,17 @@ variable "pod_mem_limit" {
 }
 
 variable "tolerations" {
-  type        = list(map(string))
-  default     = [{ key = "node-role.kubernetes.io/master", effect = "NoSchedule" }]
+  type = list(map(string))
+  default = [
+    {
+      key    = "node-role.kubernetes.io/infra",
```

Comment:
Is this something new in a new version of Kubernetes? Do all customers need it?

Personally, I used:
```
  tolerations = [
    {
      key      = null
      operator = "Exists",
      effect   = null
    }
  ]
```

This way, I'm sure I'm targeting all the current and future nodes.